.github/workflows/e2e.yml

```diff
@@ -87,3 +87,76 @@ jobs:
       with:
         name: kind_logs
         path: /tmp/kind_logs
+  e2e-install:
+    runs-on: ubuntu-20.04
+    env:
+      built_image: "numaresources-operator:ci" # Arbitrary name
+      E2E_NAMESPACE_NAME: numaresources
+      E2E_TOPOLOGY_MANAGER_POLICY: SingleNUMANodeContainerLevel
+      LOG_DIR: /tmp/test_e2e_intall_logs
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # Fetch all history for all tags and branches
+
+    - name: Set up golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
+
+    - name: Verify modules
+      run: go mod verify
+
+    - name: build test binary
+      run: |
+        make binary-e2e-rte binary-e2e-install binary-e2e-uninstall
```

Comment:
The problem is our github actions infra is already overtaxed, and I'm not convinced here that the benefits are greater than the costs. Let's keep the k8s (through kind) coverage minimal (but sufficient) until there is a crystal clear case to enhance it. Our main focus is OCP anyway.